src/chapters/sessions/managing-cookies.rst

```diff
@@ -0,0 +1,183 @@
+
+#. To set cookies, we use the (wait for it...) ``.set_cookie()`` method!
+   The general syntax for the method is:
+
+   .. sourcecode:: Python
+
+      response_name.set_cookie(cookie_name, cookie_value)
+   
+   ``response_name`` is the variable that holds the result from
+   ``make_response()``. ``cookie_name`` and ``cookie_value`` must be strings.
+
+#. The ``set_cookie()`` method can take other arguments as well. However, these
+   go beyond the scope of this course. If curious, you can find a more complete
+   description for ``set_cookie()`` `here <https://pythonise.com/series/learning-flask/flask-cookies#setting-cookies>`__.
+
+Viewing Cookies
+---------------
+
+As shown in the video, we can use our browser tools to view the cookies saved
+for a particular webpage. The specific commands will vary with each browser,
+but right-clicking on the page is a good thing to try first. Select
+*Inspect Element* from the options that pop up, and then open the *Storage*
+tab.
+
+.. figure:: figures/storage-panel.png
+   :alt: View cookies using the browser developer tools. The 'Storage' panel is selected.
+   :width: 80%
+
+   View stored cookies by using the browser tools.
+
+Note that the cookie data is NOT sent to the webpage as a value. Instead, the
+browser saves a file on our machine to keep track of the key/value pair. Even
+if we stop the Flask application, the cookie *persists*.
+
+.. figure:: figures/persistent-cookie.png
+   :alt: The storage panel shows that the cookie remains on our device.
+   :width: 80%
+
+   Even when we stop the Flask application, any cookies set by the program remain on our device.
+
+If we change the name of the cookie in our Python code and refresh the page, a
+new cookie file is saved. However, the old cookie file remains.
+
+.. figure:: figures/rename-cookie.png
+   :alt: The storage panel shows both the old and new cookie files.
+   :width: 80%
+
+   Renaming a cookie does not remove the old file.
+
+Accessing Cookie Data
+---------------------
+
+Once Flask saves cookies to our device, every request sent from our browser to
+the server includes the data from ALL of those files. We can access this data
+with the ``request`` keyword.
+
+To collect all of the cookie data, the syntax is:
+
+.. sourcecode:: Python
+
+   all_cookies = request.cookies
+
+When ``request.cookies`` executes, it returns a collection of key/value pairs
+and assigns it to ``all_cookies``. Each pair matches one of the cookie files
+stored on our device. To access the value of a specific cookie, we use bracket
+notation, ``all_cookies['cookie_name']``.
+
+To access just one of the cookie values, we use the syntax:
+
+.. sourcecode:: Python
+
+   cookie_value = request.cookies.get('cookie_name')
+
+When this statement executes, Python scans the cookie data sent with the HTTP
+request. If ``cookie_name`` matches one of the keys, then the value for that
+key is assigned to ``cookie_value``.
+
```

Comment:
This was a bit confusing - not in video and when I first copied it in I got none as the value - finally realized I need to type in fav_cookie.  

maybe change to something like this?:
 If ``cookie_name``(ie fav_cookie) matches one of the keys,

Reply:
Fixed. Thanks!